index.css

```diff
@@ -1,3 +1,102 @@
 html {
 	font-family: "ヒラギノ角ゴ Pro W3", "Hiragino Kaku Gothic Pro",Osaka, "メイリオ", Meiryo, "ＭＳ Ｐゴシック", "MS PGothic", sans-serif; 
 }
+
+body {
+	margin:0;
+}
+
+.Grid {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	grid-template-rows: repeat(15, calc(100vh / 15));
+	grid-gap: 2.5%;
+	min-height: 100vh;
+	margin:1rem;
```

Comment:
100vh + 1rem is more than 100%, so we have scrollbars.

Idea: padding instead of margin or `min-height: calc(100vh - 2rem)`.